src/utils.c

```diff
@@ -157,33 +157,36 @@ gint utils_get_line_endings(const gchar* buffer, gsize size)
 
 gboolean utils_isbrace(gchar c, gboolean include_angles)
 {
+	return utils_is_opening_brace(c, include_angles)
+		|| utils_is_closing_brace(c, include_angles);
```

Comment:
`||` should go at the end of the previous line (yeah I know there are some exceptions to this in the current code, but lets not add more :)